docs/tremor-script/index.md

```diff
@@ -167,6 +167,59 @@ Record literals in `tremor-script` are syntactically equivalent to JSON document
 }
 ```
 
+#### Binary
+
+Binaries are based on the [Erlang bit syntax](https://erlang.org/doc/programming_examples/bit_syntax.html).
+
+Binary grammar:
+
+> ![record grammar](grammar/diagram/Binary.png)
+
+Binaryfields grammar:
+
+> ![fields grammar](grammar/diagram/BinaryFields.png)
+
+Binaryfield grammar:
+
+> ![field grammar](grammar/diagram/BinaryField.png)
+
+
+Parts of each field are: `<value>:<size>/<type>` where both `size` and `type` are optional.
+
+The types currently supported are:
+
+* `binary` - this can handle both binaries and strings, `size` here refers to the number of bytes
+* `integer` - this can represent integers, `size` here means size in bits. In addition the type can be specified with `big` and `little` for indianness and `singed` and `unsigned` for singedness.
```

Comment:
```suggestion
* `integer` - this can represent integers, `size` here means size in bits. In addition the type can be prefixed with `big` and `little` for indianness and `singed` and `unsigned` for singedness.
```

Reply:
Can you add some simple examples on what is possible?

e.g.:

```<<1:1, 42:7>>```

```<< (1 + 1)/unsigned-big-integer>>```

especially literals as field value and different possible types with prefixes.

Reply:
Also the RFC lists the default being `:8/big-unsigned-integer`, which might also be useful to list here.

Reply:
:+1: